saviSimulation/savi/jason_processing/UxVBehavior.java

```diff
@@ -210,7 +210,7 @@ protected void updatePercepts(PVector mypos) {
 	 * Makes with a random probability, a random error on the UxV position perceived
 	 */
 	protected PVector getPositionWithError(PVector position, double sensorErrorProb) {
-		PVector positionWithError = new PVector();
+		PVector positionWithError = position.copy();
```

Comment:
Another fix. This function will return 0, 0, 0 as the position perception when the sensor doesn't fail. This is fixed by initializing the PVector with the current position.

Reply:
Dealt with in another PR #136